src/store/reducers/grantsReducer.js

```diff
@@ -18,7 +18,7 @@ import {
 
 const initialState = {
   grants: [],
-  profileGrants: [],
+  applicantGrants: [],
```

Comment:
I think this name change will help with the readability and clarity, good choice!